pim/utils.py

```diff
@@ -149,10 +150,10 @@ class spinner(threading.Thread):
     """
     def __init__(self):
         if os.name == 'posix':
-            self.chars = (unicodedata.lookup('FIGURE DASH'),u'\\ ',u'| ',u'/ ')
```

Comment:
So I guess, in conclusion, it is probably fine as it is since you are going to pull it out anyway